README.md

```diff
@@ -22,3 +22,63 @@ Trello board: https://trello.com/b/vptWzBnE/generate-privacy-statements-portage
 - If you want to use Google Analytics or to send feedback or link emails, you have to set up proper environment variables. Copy the `.env.example` file into `.env` and set the variables appropriately.
 - Next install the third party modules using `npm install`
 - You can now run the app locally! run `npm run dev` and then open a web browser to `localhost:3000`.
+
+# April 2023 Update
+
+## Running it locally
+At the moment, it is still possible to run locally, one way is simply running `npm run dev`
+
+
+As part of the Migration to AWS from Heroku, infrastructure for this was created and is hosted on AWS.
+
+## Requires
+- Terraform (https://www.terraform.io/)
+- Terragrunt (https://terragrunt.gruntwork.io/)
+
+## How is this repository structured?
+
+The Terraform code contained in `aws` is split into several independent modules that all use their own remote Terraform state file.These modules know nothing about Terragrunt and are used by Terragrunt as simple infrastructure definitions.
+
+The directory structure inside `aws` reflects the split into independent modules. For example, `ecr`, contains the lifecycle policy and configuration that creates the ECR that will be used by the `app (lambda)` module.
+
+At the moment, there is only a production environment in the `env` which contains the Terragrunt scripts.
+
+## What is each Terraform module
+
+`aws/app`
```

Comment:
What would you think of making these each a level three heading and linking to the specific module?
```suggestion
### [`aws/app`](terragrunt/aws/app)
```